Appropriation Art Exhibit/exhibit.py

```diff
@@ -0,0 +1,697 @@
+__author__ = 'Harriet'
+
+import random
+import math
+import os
+
+from PIL import Image
+
+
```

Comment:
Comments are clear, and with some reading through I can see what is going on. Though some sections may be beyond my current understanding.